assembly/Int/CompactInt.ts

```diff
@@ -62,8 +62,9 @@ export class CompactInt implements Codec {
      * @description Instantiates Compact Int from u8[] SCALE encoded bytes
      * Compact Int decodes int8, int16, int32, int64 size correctly  
      */
-    static fromU8a (value: u8[]): CompactInt {
-        const decodedData = Bytes.decodeCompactInt(value);
+    static fromU8a (value: u8[], index: i32 = 0): CompactInt {
+        assert(value.length > 0, "CompactInt: Empty bytes array provided");
```

Comment:
shouldn't we add that value.length - index >= 0 ? Same on the other asserts as-well